metadata-translation/notebooks/changesheets-example.ipynb

```diff
@@ -1066,10 +1066,189 @@
+      "{\n",
+      "  \"id\": \"gold:Gs0114675\",\n",
+      "  \"name\": \"Deep subsurface shale carbon reservoir microbial communities from Ohio and West Virginia, USA\",\n",
+      "  \"description\": \"Despite the importance of carbon-rich deep shale systems, little is known about microbe-catalyzed processes that may occur naturally within these formations, and how these change as a result of engineered activities. This has important implications on biodiversity change within the deep terrestrial biosphere, the efficiency of energy recovery, and the sustainability of our energy infrastructure. Although limited studies have identified diverse active microorganisms in deep shale formations, more research is needed to understand microbial mechanisms for growth and survival, and how such activities impact carbon and other biogeochemical cycles.\",\n",
+      "  \"ecosystem\": \"Environmental\",\n",
+      "  \"ecosystem_category\": \"Terrestrial\",\n",
+      "  \"ecosystem_type\": \"Deep subsurface\",\n",
+      "  \"ecosystem_subtype\": \"Unclassified\",\n",
+      "  \"specific_ecosystem\": \"Unclassified\",\n",
+      "  \"principal_investigator\": {\n",
+      "    \"has_raw_value\": \"Kelly Wrighton\"\n",
+      "  },\n",
+      "  \"doi\": {\n",
+      "    \"has_raw_value\": \"10.25585/1487763\"\n",
+      "  },\n",
+      "  \"type\": \"nmdc:Study\"\n",
+      "}\n",
+      "------------------ AFTER ------------------\n",
+      "{\n",
+      "  \"id\": \"gold:Gs0114675\",\n",
+      "  \"name\": \"Deep subsurface shale carbon reservoir microbial communities from Ohio and West Virginia, USA\",\n",
+      "  \"description\": \"Despite the importance of carbon-rich deep shale systems, little is known about microbe-catalyzed processes that may occur naturally within these formations, and how these change as a result of engineered activities. This has important implications on biodiversity change within the deep terrestrial biosphere, the efficiency of energy recovery, and the sustainability of our energy infrastructure. Although limited studies have identified diverse active microorganisms in deep shale formations, more research is needed to understand microbial mechanisms for growth and survival, and how such activities impact carbon and other biogeochemical cycles.\",\n",
+      "  \"ecosystem\": \"Environmental\",\n",
+      "  \"ecosystem_category\": \"Terrestrial\",\n",
+      "  \"ecosystem_type\": \"Deep subsurface\",\n",
+      "  \"ecosystem_subtype\": \"Unclassified\",\n",
+      "  \"specific_ecosystem\": \"Unclassified\",\n",
+      "  \"principal_investigator\": {\n",
+      "    \"has_raw_value\": \"Kelly Wrighton\"\n",
+      "  },\n",
+      "  \"doi\": {\n",
+      "    \"has_raw_value\": \"10.25585/1487763\"\n",
+      "  },\n",
+      "  \"type\": \"nmdc:Study\",\n",
+      "  \"has_credit_associations\": [\n",
+      "    {\n",
+      "      \"applied_role\": \"Conceptualization\"\n",
+      "    },\n",
```

Comment:
this is incorrect. There should be one dictionary in the has_credit_associations list, not two. The one dictionary should have both applied_role and applied_to_person keys.

Reply:
Nice catch ... I need to merge those two dicts.